src/tools/line.js

```diff
@@ -0,0 +1,166 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.tools.line = function() {
+
+        var xScale = d3.time.scale(),
+            yScale = d3.scale.linear(),
+            value = fc.utilities.fn.identity,
+            keyValue = fc.utilities.fn.index,
+            label = value,
+            // specifies whether the value is domain or screen coordinates
```

Comment:
On second thoughts, the fix is just a ternary operator in the x/yScale calls in crosshairs. Maybe it can be fixed as part of this?